tests/test_boards.py

```diff
@@ -778,3 +780,113 @@ def test_energenie():
         pins[5].assert_states_and_times([(0.0, False), (0.1, True), (0.25, False)])
         device1.close()
         assert repr(device1) == '<gpiozero.Energenie object closed>'
+
+def test_statuszero_init():
+    with StatusZero() as sz:
+        assert sz.namedtuple._fields == ('one', 'two', 'three')
+    with StatusZero('a') as sz:
+        assert sz.namedtuple._fields == ('a',)
+    with StatusZero('a', 'b') as sz:
+        assert sz.namedtuple._fields == ('a', 'b')
+    with StatusZero('a', 'b', 'c') as sz:
+        assert sz.namedtuple._fields == ('a', 'b', 'c')
+    with pytest.raises(ValueError):
+        StatusZero('a', 'b', 'c', 'd')
+    with pytest.raises(ValueError):
+        StatusZero('0')
```

Comment:
Should these 'invalid device identifier' tests have been added to the CompositeDevice tests, rather than to just StatusZero and StatusBoard?